cmd/workspace.go

```diff
@@ -166,10 +189,29 @@ func workspaceUserAdd(cmd *cobra.Command, args []string) error {
 		// fmt.Println("Default workspace id not set, set default workspace id or pass a workspace in via the --workspace-id flag")
 	}
 
+	if err := validateRole(role); err != nil {
+		return errors.Wrap(err, "failed to find a valid role")
+	}
+
 	// Silence Usage as we have now validated command input
 	cmd.SilenceUsage = true
+	return workspace.Add(ws, args[0], role)
+}
+
+func workspaceUserUpdate(cmd *cobra.Command, args []string) error {
+	ws, err := coalesceWorkspace()
+	if err != nil {
+		return errors.Wrap(err, "failed to find a valid workspace")
+		// fmt.Println("Default workspace id not set, set default workspace id or pass a workspace in via the --workspace-id flag")
```

Comment:
should we delete this commented out line?

Reply:
done.